src/main/java/uk/gov/hmcts/reform/et/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -57,10 +57,11 @@ public static Boolean isHearingIsLinkedFlag(CaseData caseData) {
     }
 
     public static String getTribunalAndOfficeLocation(CaseData caseData) {
-        return defaultString(caseData.getCaseManagementLocationCode(),
-                Optional.ofNullable(caseData.getCaseManagementLocation())
-                        .map(CaseLocation::getBaseLocation)
-                        .orElse(" ")
+        return defaultString(
+            caseData.getCaseManagementLocationCode(),
+            Optional.ofNullable(caseData.getCaseManagementLocation())
+                .map(CaseLocation::getBaseLocation)
+                .orElse(" ")
```

Comment:
Why this change?

Reply:
No reason. Intellij just autoformatted. I had been making changes in this file but reverted.

Reply:
Could you revert plz? Subjective but I think this looks worse, I wouldn't ask but it looks like you've got changes to make regarding a new cve anyway